nncf/tensorflow/utils/state.py

```diff
@@ -86,3 +87,48 @@ def deserialize(self, string_value: str) -> None:
         :param string_value: A serialized compression state.
         """
         self._state = json.loads(string_value)
+
+
+class ConfigState(tf.train.experimental.PythonState):
+    """
+    Used to save/load a config into the tf.train.Checkpoint.
+    """
+
+    def __init__(self, config: Optional[Dict[str, Any]] = None):
+        """
+        :param config: Config.
+        """
+        self.config = config
+
+    def serialize(self) -> str:
+        """
+        Callback to serialize the config.
+
+        :return: A serialized config.
+        """
+        data = {
+            "quantization": {
+                "quantizer_setup": TFQuantizationSetup.from_state(self.config["quantizer_setup"]).get_state(),
+            }
+        }
```

Comment:
It is necessary to unwrap TensorFlow wrappers for lists and dictionaries into built-in types.